public/index.html

```diff
@@ -8,9 +8,19 @@
     <meta name="description" content="Anonymous Team Feedback" />
 
     <link rel="icon" type="image/png" sizes="96x96" href="/favicon.png" />
-    <link href="https://fonts.googleapis.com/css?family=Miriam+Libre&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Baloo+Thambi&display=swap" rel="stylesheet">
```

Comment:
These fonts are no longer needed?

Reply:
they are, those are the fonts we're using for the application.